### PR TITLE
Fix example of using `composed_of`

### PR DESCRIPTION
**Pull request by bogdanvlviv**
- Fix mapping of attributes in the example

See also https://github.com/rails/rails/blob/23226d04f921b79f0077ba38c5a5a923b6d43f89/activerecord/lib/active_record/aggregations.rb#L176-L183

### CODE REVIEW

File: activerecord/lib/active_record/aggregations.rb [hunk 1]
```diff
@@ -35,7 +35,7 @@ def init_internals
       # the database).
       #
       #   class Customer < ActiveRecord::Base
-      #     composed_of :balance, class_name: "Money", mapping: %w(amount currency)
+      #     composed_of :balance, class_name: "Money", mapping: [ %w(balance_amount amount), %w(balance_currency currency) ]
```
[the remark this answers is not in the document]
Comment (bogdanvlviv) [reply activerecord/lib/active_record/aggregations.rb]: [about :mapping](https://github.com/rails/rails/blob/011f76e57b145b9e5ef1fd520df7d7096cf8896a/activerecord/lib/active_record/aggregations.rb#L193):
:mapping - Specifies the mapping of entity attributes to attributes of the value object. <b>Each mapping is represented as an array where the first item is the name of the entity attribute and the second item is the name of the attribute in the value object</b>. The order in which mappings are defined determines the order in which attributes are sent to the value class constructor.


so, if extend `BugTest#test_association_stuff` in https://gist.github.com/vipulnsward/366abfd6bfcb05f39de090420c0d829e
```ruby
assert_equal 20, customer.amount
# Expected: 20
#   Actual: "USD"

assert_equal 'USD', customer.currency
# Expected: "USD."
#   Actual: nil
```
it'll fail.

[the remark this answers is not in the document]
Comment (kamipo) [reply activerecord/lib/active_record/aggregations.rb]: It was broken by #23256.
Just #23256 should be reverted.

[the remark this answers is not in the document]
Comment (kamipo) [reply activerecord/lib/active_record/aggregations.rb]: Reverted in b2c5ec234293813d16a65f73ee688a18900bf173.

**Comment by kamipo** [comment]
> The following example # finds all customers with +balance_amount+ equal to 20 and +balance_currency+ equal to "USD":

This example was added in abdf546ad6d02ecb95766e73cd3c645a48c954de, but it was inconsistent with `composed_of :balance` definition in the `Customer`.

@bogdanvlviv Can you create a new PR that changing the example using another `composed_of` attribute like `Customer.where(address: Address.new(...))`?

**Pull request closed without merging**